ui/raidboss/data/05-shb/raid/e6s.js

```diff
@@ -23,10 +23,15 @@
     },
     {
       id: 'E6S Ferostorm',
-      regex: Regexes.startsUsing({ source: 'Garuda', id: '4BF[EF]', capture: false }),
+      regex: Regexes.startsUsing({ source: ['Garuda', 'Raktapaksa'], id: ['4BF[EF]', '4C0[45]'], capture: false }),
+      regexDe: Regexes.startsUsing({ source: ['Garuda', 'Raktapaksa'], id: ['4BF[EF]', '4C0[45]'], capture: false }),
+      regexFr: Regexes.startsUsing({ source: ['Garuda', 'Raktapaksa'], id: ['4BF[EF]', '4C0[45]'], capture: false }),
+      regexJa: Regexes.startsUsing({ source: ['ガルーダ', 'ラクタパクシャ'], id: ['4BF[EF]', '4C0[45]'], capture: false }),
+      regexCn: Regexes.startsUsing({ source: '迦楼罗', id: '4BF[EF]', capture: false }),
+      regexKo: Regexes.startsUsing({ source: '가루다', id: '4BF[EF]', capture: false }),
```

Comment:
Korea is still 5.0 so we don't know the official translation yet, literally translation is 락타팍사.
```suggestion
      regexKo: Regexes.startsUsing({ source: ['가루다', '락타팍사'], id: '4BF[EF]', capture: false }),
```